plugins/webp-uploads/picture-element.php

```diff
@@ -23,13 +23,7 @@ function webp_uploads_wrap_image_in_picture( string $image, string $context, int
 		return $image;
 	}
 
-	$file = get_attached_file( $attachment_id, true );
-	// File does not exist.
-	if ( false === $file || ! file_exists( $file ) ) {
```

Comment:
Related to the above, the `file_exists()` check was never needed here anyway, so this is okay to be removed.